NDT7.xcodeproj/project.pbxproj

```diff
@@ -147,6 +146,7 @@
 		88E34DA3226E720900067D4F /* Constants.swift in Sources */ = {isa = PBXBuildFile; fileRef = 88E34DA1226E720900067D4F /* Constants.swift */; };
 		88E34DA4226E720900067D4F /* Constants.swift in Sources */ = {isa = PBXBuildFile; fileRef = 88E34DA1226E720900067D4F /* Constants.swift */; };
 		88E34DA5226E720900067D4F /* Constants.swift in Sources */ = {isa = PBXBuildFile; fileRef = 88E34DA1226E720900067D4F /* Constants.swift */; };
+		9CD7B7FE254B1502004E9A7D /* Asurion-Private-NDT7.podspec in Resources */ = {isa = PBXBuildFile; fileRef = 9CD7B7FD254B1502004E9A7D /* Asurion-Private-NDT7.podspec */; };
```

Comment:
Please, don't modify the pod spec, I can create a new release and update the pod.